Examples/Algorithms/TrackFinding/src/SeedingAlgorithm.cpp

```diff
@@ -105,7 +108,16 @@ ActsExamples::ProcessCode ActsExamples::SeedingAlgorithm::execute(
   auto topBinFinder = std::make_shared<Acts::BinFinder<SimSpacePoint>>(
       Acts::BinFinder<SimSpacePoint>());
   auto grid = Acts::SpacePointGridCreator::createGrid<SimSpacePoint>(m_gridCfg);
-  auto spacePointsGrouping = Acts::BinnedSPGroup<SimSpacePoint>(
+
+  auto get_grid = std::get<Acts::detail::Grid<std::vector<std::unique_ptr<
+                     const Acts::InternalSpacePoint<SimSpacePoint>>>,
+                 Acts::detail::Axis<Acts::detail::AxisType::Equidistant,
+                              Acts::detail::AxisBoundaryType::Closed>,
+                 Acts::detail::Axis<Acts::detail::AxisType::Variable,
+                              Acts::detail::AxisBoundaryType::Bound>>
+                              >(*grid);
+
+  auto spacePointsGrouping = Acts::BinnedSPGroup<SimSpacePoint, Acts::detail::AxisType::Variable>(
```

Comment:
So if I'm not thinking about it wrong, all of this would go inside an `std::visit`, which then gives you the concrete grid type without having to  write `Variable` here. Then you execute the seed finder created from this grid below, and will the `SimSeedContainer` all inside the `std::visit`. After that, you should have a uniform result type that you can process outside the visit. 

Am I missing something?